### ISSUE TITLE
In PHPBB 3.2 to create the entries does not work

### ISSUE THREAD

**Issue by AlexL777** [pierrdu/lmdi_gloss#1]
Hello Developers,

have you tested this extension already under PHPBB 3.2. I have with me PHPBB 3.2 Alpha 1 and have the extension installed like. But when I go to Entry Create, nothing happens.

Would that be possible to use this extension to PHPBB 3.2? Yet I have seen on the server PHP 7.0.

Thank you in advance.

**Comment by pierrdu** [comment]
Not yet. I'm waiting for a beta version to do it.

**Comment by pierrdu** [comment]
I checked with 3.2.a2 and I get a lot of messages:
[phpBB Debug] PHP Notice: in file [ROOT]/phpbb/request/request.php on line 78: Object of class phpbb/request/deactivated_super_global could not be converted to int

**Comment by pierrdu** [comment]
It seems that the root of all problems is the use of request_var, to be replaced by $request->variable.

**Comment by AlexL777** [comment]
Ok, then we are still awaiting the beta version. This is perhaps the better.

**Comment by pierrdu** [comment]
I'll keep you informed

**Comment by AlexL777** [comment]
OK thank you :)

**Comment by pierrdu** [comment]
It now works under 3.2.0.alpha2

**Comment by AlexL777** [comment]
Thank you.

Unfortunately I can not update to PHPBB 3.2 Alpha 2, because the update system will generate an error and can be found on GitHub unfortunately not create Issues.

So I now have PHPBB 3.2 Alpha 1 and when I'm in the settings of this extension, we displayed when you save this error.

    Fatal error: Call to a member function set() on null in /www/htdocs/xxxxxxx/cms/phpbb32/includes/functions_compatibility.php on line 299

Can you do it again you please view.

Thank you in advance.

**Comment by pierrdu** [comment]
Thanks for your patience.
I can't do anything about this error because it happens in a core phpbb file (functions_compatibility). On the other hand, I've found that the options of the extension don't get saved to the config table. You're using 3.2.0.alpha1?

**Comment by AlexL777** [comment]
Yes, I use PHPBB 3.2 Alpha 1. On alpha2 I wanted to upgrade, but the update system also generates an error. I wanted to report on GitHub, but you can not create Issues. This leaves only the support forum.

**Comment by pierrdu** [comment]
Well, we'll have to let some time elapse before going further.

**Comment by pierrdu** [comment]
Could you test with version b2?
For me, it works.

**Comment by AlexL777** [comment]
Hello,
I have this extension installed again and actually it looked so good, but when you create a new item in the Glossary me this error message.

    Allgemeiner Fehler
    SQL ERROR [ mysqli ]

    Unknown column 'cat' in 'field list' [1054]

    SQL

    INSERT INTO phpbb_glossary (variants, term, description, cat, ilinks, elinks, label, picture, lang) VALUES ("PHP Fusion", "PHP Fusion", "PHP Fusion ist ein passables OpenSource und Portal CMS mit Blog, Forum, FAQs, Downloads, Weblinks, News und Artikeln. Die Version 9 wird gerade entwickelt und befindet sich im RC-Stadium.", "", "", '', "", "nopict.jpg", "en")

    BACKTRACE

Apparently something with the database.

Would be helpful if you could help me. Thanks in advance.

**Comment by pierrdu** [comment]
You probably didn't install the latest version. Uninstll it, delete thedata and install the latest version.
